pkgs/applications/virtualization/xen/generic.nix

```diff
@@ -629,11 +629,11 @@ stdenv.mkDerivation (finalAttrs: {
     longDescription =
       # Starts with the longDescription from ./packages.nix.
       (packageDefinition.meta.longDescription or "")
-      +
-        lib.strings.optionalString (!withInternalQEMU)
-          "\nUse with `qemu_xen_${lib.stringAsChars (x: if x == "." then "_" else x) branch}`"
-      + lib.strings.optionalString latest "or `qemu_xen`"
-      + "."
+      + lib.strings.optionalString (!withInternalQEMU) (
```

Comment:
This seems to not change anything except formatting? Or am I misreading?

Reply:
There's an extra pair of parenthesis, to only add the period and the newline at the end if `withInternalQEMU` is false.